xpk.py

```diff
@@ -3187,18 +3224,25 @@ def directory_path_type(value):
 
 ### Required arguments
 workload_delete_parser_required_arguments.add_argument(
+    '--cluster',
+    type=str,
+    default=None,
+    help='The name of the cluster to delete the job on.',
+    required=True,
+)
+
+### Optional arguments
```

Comment:
Can you add a `--force` argument that skips the user prompt? This is good for automated tests for example. Or if a user really believes in themselves. You can build from https://github.com/google/xpk/blob/main/xpk.py#L2823-L2830

Reply:
Sure, added!